src/serde_json.rs

```diff
@@ -20,7 +20,7 @@ fn json_value_to_value(value: &serde_json::Value) -> EvaluationResult<Value> {
     match value {
         serde_json::Value::Bool(value) => Ok(Value::Bool(*value)),
         serde_json::Value::Number(value) if value.is_i64() => {
-            Ok(Value::Int(value.as_i64().unwrap()))
+            Ok(Value::Int(value.as_i64().unwrap())) //TODO: remove unwrap or use expect
```

Comment:
can you explain the TODO? how do you wan to remove the `unwrap()`?

Reply:
In general, it is better to avoid any potential panic in libraries, so in my opinion, the error could be returned to the caller as EvaluationResult::Err(...), or when any error is impossible then `expect` will make it clearer to anyone who gets an unexpected panic.